### ISSUE TITLE
Unable to display the search box, or unable to get search to work: on sharpedge CMS V 3.42.76 

### ISSUE THREAD

**Issue by ebonylegg** [srpurdy/SharpEdgeCMS#40]
I have tried to get the search to work, both by going to the the url htt://website.com/en/search as well as attempting to add the search as a widget or module in the side bar or content containers. Nothing happens except the message telling that search requires 3 or more characters.

**Comment by srpurdy** [comment]
<?php $search_attr = array('class' => 'custom-css-class');?>
<?php echo form_open('search', $search_attr);?>
<input type="text" placeholder="Search" class="search-query ciph_search" name="search" value="" />
<input type="submit" class="btn btn-primary" value="Search" />
<?php echo form_close();?>

You can either create a file based widget or put this directly in your theme template.

**Comment by srpurdy** [comment]
The other option is you create a hidden input manually in your code you can add a line like

<input type="hidden" name="csrf_sharpedgeV320" value="<?php echo $this->security->get_csrf_hash() ?>" id="csrf_protection" />

the id= you can use this for custom ajax requests to extract the CSRF token.

**Comment by ebonylegg** [comment]
hehehe ouch, now you have gone beyond my technical know how. I am new to this, and I am just starting to make heads and tails of this. If you are not too busy,  kindly Implement the search for me, and I will just download the new template file and FTP the changes to my website. I would not really know where to begin. The last time I tried creating a module or a widget, I got a blank home page and it too a while to restore it. lol.
